components/ArticleList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"; 
+import ArticleCard from "./ArticleCard"
+
+
+const ArticleList= () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [list , setList] = useState([]);
+
+
+
+    useEffect(() => {
+        console.log("useEffect triggered!")
+        fetch("https://nc-news-zcdp.onrender.com/api/articles")
+        .then((response) => response.json())
```

Comment:
It would be ideal to extract this request to the api call out into it's own file called `api.js` or similar so that we're separating our concerns. Ideally, you'd write a function that handles requesting the articles, which you can then require in and use here. Also, you might find `axios` a more convenient alternative to fetch (especially when your requests get a bit more complex)!